.setup/bin/sample_courses/__init__.py

```diff
@@ -26,20 +26,21 @@ def parse_args():
     parser.add_argument("--courses_path", default=os.path.join(SETUP_DATA_PATH, "courses"),
                         help="Path to the folder that contains .yml files to use for course creation. Defaults to "
                              "../data/courses")
-    parser.add_argument("--install_dir", type=str, default=SUBMITTY_INSTALL_DIR, help="install path of submitty")
-    parser.add_argument("--data_dir", type=str, default=SUBMITTY_DATA_DIR, help="data path of submitty")
```

Comment:
Changed the default to what the default value of the variable was because this function was moved to __init__.py, to make it easier to see how each variable is being set